integration-tests/drivers/TestDriver063/.gitignore

```diff
@@ -34,6 +34,7 @@ local.properties
 node_modules/
 npm-debug.log
 yarn-error.log
+package-lock.json
```

Comment:
Should this actually be in gitignore? I always thought package-lock.json was supposed to be in version control, at least based on things I've read in SO.

Reply:
With the test project in particular, I've been struggling with `package-lock.json`.

Right now, my local `package-lock.json` has the following definition:

```
    "@heap/react-native-heap": {
      "version": "file:../../heap-react-native-heap-0.17.0.tgz",
      "integrity": "sha512-KhBins/aSATHvr3SJosfh0VbQ4wsBeWko4dRStQx9RfjQCCcxzBgnjX3hLNrLNtYZzOJ8aYYZfuLlk8M8U66/g==",
      "requires": {
        "@babel/core": "^7.16.0",
        "@types/react-reconciler": "^0.26.4",
        "babel-plugin-add-react-displayname": "0.0.5",
        "babel-template": "^6.26.0",
        "babel-types": "^6.26.0",
        "flat": "^5.0.2",
        "hoist-non-react-statics": "^3.3.2"
      }
    },
```

If I make a change to one of our source files, repack the project, and wipe out `node_modules`, running `npm install` will fail to install my updated local source and instead install a cached version of `react-native-heap` that has that sha512 signature.

There may be a better way to force the install despite the changed signature, but deleting `package-lock.json` was my solution to get the tests to consistently consume the latest source.

Reply:
I found an alternative solution with:

```sh
npm uninstall @heap/react-native-heap || true
npm install ../../heap-react-native-heap-*.tgz